[SOAR-1155, SOAR-768] Fortinet Simplify Fortinet Address Object actions and add whitelist WIP (#481)
rapid7/insightconnect-plugins: fortinet_fortigate/help.md

|Name|Type|Default|Required|Description|Enum|Example|
|----|----|-------|--------|-----------|----|-------|

> |host|string|None|True|This can be an IP-Address, an IP CIDR or a domain name. e.g. 192.168.1.1, 192.168.1.0/24, google.com|None|198.51.100.100|

Reviewer: ```suggestion
|host|string|None|True|This can be an IP-Address, an IP CIDR or a domain name|None|198.51.100.100|
```

Reviewer 2: I think this is fine, original has more detail

|name|string|None|False|Optional name to give this address object. If not provided, the name will be the IP address or domain name|None|FriendlyName|
|whitelist|[]string|None|False|This list contains a set of network object that should not be blocked. This can include IPs, CIDR notation, or domains|None|[198.51.100.100, google.com, 192.0.2.0/24]|

Example input:

```
{
  "host": "192.168.7.7",
  "name": "testing",

>   "whitelist": ["198.51.100.100", "google.com", "192.0.2.0/24"]

Reviewer 2: ```suggestion
  "whitelist": ["198.51.100.100", "example,com", "192.0.2.0/24"]
```

|Name|Type|Default|Required|Description|Enum|Example|
|----|----|-------|--------|-----------|----|-------|
|host|string|None|True|The host. This can be an IP an IP CIDR e.g. 198.51.100.0/24 or a domain name|None|198.51.100.100|

Example input:

```

> "host": "198.51.100.100"

Reviewer 2: This needs to be valid JSON

# Version History

> * 2.0.0 - Simplify Fortinet Address Object actions | Add whitelist safety check to Create Address Object action

Reviewer 2: ```suggestion
* 2.0.0 - Simplify the Create Address Object action to auto-detect the input type | Add whitelist safety check to Create Address Object action
```
